rwengine/src/engine/GameWorld.cpp

```diff
@@ -76,9 +76,13 @@ GameWorld::GameWorld(Logger* log, GameData* dat)
 }
 
 GameWorld::~GameWorld() {
-    for (auto& p : allObjects) {
-        delete p;
-    }
+    // Bullet requires to remove each object before all physic world
```

Comment:
This should already happen, because the object pool members in GameWorld are before dynamicsWorld.

Reply:
I remember some some asan issue. Will retest it. ;)

Reply:
ahh, there's problem with `TrafficDirector` in test `test_available_nodes`.